ocaml/forkexecd/lib/forkhelpers.ml

```diff
@@ -357,10 +362,12 @@ let execute_command_get_output_inner ?env ?stdin ?(syslog_stdout = NoSyslogging)
     )
     (fun () -> List.iter Unix.close !to_close)
 
-let execute_command_get_output ?env ?(syslog_stdout = NoSyslogging)
+let execute_command_get_output ?traceparent ?env ?(syslog_stdout = NoSyslogging)
```

Comment:
Is `traceparent` written this way in other code already? Otherwise I would prefer a better spelling.

Reply:
There are some places that call the parent span `traceparent`, some that call it `parent`, some that call it `tracing`. Given the function used is called `with_tracing` is it more preferable to call it `tracing`?